modules/@ergonode/authentication/src/components/Forms/PasswordRecoveryCheckEmail.vue

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<template>
+    <LoginForm
+        :submit-title="$t('authentication.forms.checkEmail.redirectTo')"
+        @submit="onRedirectToLogin">
+        <template #header>
+            <div class="recovery-form__header">
+                <Fab @click.native="onBack">
+                    <template #icon="{ color }">
+                        <IconArrowPointer :fill-color="color" />
+                    </template>
+                </Fab>
+                <h2 v-text="$t('authentication.forms.checkEmail.title')" />
```

Comment:
Form has title - i would create component for h2 and call it FormTitle and replace it everywhere